src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge17.java

```diff
@@ -0,0 +1,36 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static org.owasp.wrongsecrets.RuntimeEnvironment.Environment.DOCKER;
+
+@Component
+@Order(17)
+public class Challenge17 extends Challenge {
+
+    public Challenge17(ScoreCard scoreCard) {
+        super(scoreCard);
+    }
+
+    @Override
+    public Spoiler spoiler() {
+        return new Spoiler("this is temporary and is not hardcoded anywhere");
+    }
+
+    @Override
+    public boolean answerCorrect(String answer) {
+        return "this is temporary and is not hardcoded anywhere".equals(answer);
```

Comment:
do we want to parse the actual file for the content with a grep? or just hide this :)
hiding it has a few steps:
1. generate the answer you want
2. in debug mode: encrypt the data with the given key used in other challenges
3. store the encrypted answer.
4. add required tests

Reply:
I have used a randomly generated password which replaces the placeholder now using SED. What do you think?